src/lib/logger.ts

```diff
@@ -122,9 +122,7 @@ interface IDocumentInfo {
 
 interface ITeacherNetworkInfo {
   networkClassHash?: string;
-  networkClassName?: string;
-  networkUserId?: string;
-  networkUserName?: string;
+  networkUsername?: string;
```

Comment:
Looking at the code, I'm wondering if it might be simpler/more consistent to keep this as `networkUserId` which is just the id, and then it becomes the logger's responsibility to append the `@portal`, which is the way the regular user id/name are handled. Then the only external clients that would need to worry about appending the `@portal` would be clients that aren't passing `ITeacherNetworkInfo`, like the `<CollapsibleDocumentsSection>` above.

Reply:
I made some modifications.  Should be more in line with your comments.